content/automate/ManageIQ/System/Event/EmsEvent/LenovoXclarity.class/__class__.yaml

```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: 
+    name: LenovoXclarity
```

Comment:
I think the only way would be adding dependency to this PR because they are different projects, content and core. Have any way to add the change into event parser here?